test/components/DashboardTest.js

```diff
@@ -9,7 +9,7 @@ var DashboardConfigurationStore = require('../../src/stores/DashboardConfigurati
 describe('Dashboard', function () {
     it('loads the configuration from the store', function () {
         TestUtils.renderIntoDocument(<Dashboard />);
-        expect(DashboardConfigurationStore.get.mock.calls.length).toEqual(1);
+        expect(DashboardConfigurationStore.get.mock.calls.length).toEqual(0);
```

Comment:
This is asserting the wrong thing. If this assertion is no longer relevent to the current functionality then delete the assertion. Otherwise a fix will need to be made.